Source/Core/Core/Core.cpp

```diff
@@ -893,7 +893,7 @@ void UpdateTitle()
 {
   float FPS = g_perf_metrics.GetFPS();
   float VPS = g_perf_metrics.GetVPS();
-  float Speed = g_perf_metrics.GetSpeed();
+  float Speed = 100.0 * g_perf_metrics.GetSpeed();
```

Comment:
This is pre-existing, but... all of these getters return doubles, why are they being converted to float for display?

Reply:
good point